spec/features/slot_ranges_spec.rb

```diff
@@ -69,15 +69,17 @@ def when_they_save_the_users_time_slots
   end
 
   def then_they_are_told_that_the_guider_has_been_created
-    expect(page).to have_content "Guider \"#{@guider.name}\" has been created"
+    slot_range = @guider.slot_ranges.first
+    expect(page).to have_content "Slot Range \"#{slot_range.display_title}\" has been created"
```

Comment:
I think this is being over-specific. Can we just use a page object and check for the presence of an element instead? This is stuff that changes so these style of matchers end up being super-brittle.